arch/arm64/Kconfig

```diff
@@ -352,6 +366,7 @@ config ARM64_DECODEFIQ
 config ARM64_GIC_VERSION
 	int "GIC version"
 	default 2           if ARCH_CHIP_A64
```

Comment:
wrong alignment, you can move this to the previous line `default 2           if ARCH_CHIP_A64 | ARCH_CHIP_BCM2711`